src/configs/status.rs

```diff
@@ -17,6 +17,7 @@ pub struct StatusConfig<'a> {
     pub pipestatus: bool,
     pub pipestatus_separator: &'a str,
     pub pipestatus_format: &'a str,
+    pub pipestatus_segment_format: &'a str,
```

Comment:
`Option<&'a str>` might be better here to make it more obvious that this is optional.